visualizeInput.py

```diff
@@ -0,0 +1,31 @@
+# To visualise the data from Hacktoberfest_Inputt, we plot the occcurences of 
+# each relation
+
+import pandas as pd
+import matplotlib.pyplot as plt
+from pprint import pprint
+from collections import Counter
+
```

Comment:
```
def visualize(user_input):
    df = pd.read_excel(user_input).to_dict()
```

Eliminate hard coded file, make it modular!

Reply:
Thanks for the review😉.